src/app/Main.java

```diff
@@ -65,4 +67,59 @@ public static void main(String[] args) {
         application.pack();
         application.setVisible(true);
     }
-}
\ No newline at end of file
+
+    public static void main2(String[] args) {
+        //To read input from console
+        Scanner scanner = new Scanner(System.in);
+
+        //Prompt user
+        System.out.println("Sign up or log in? 1 for sign up, 2 for log in.");
```

Comment:
Another way to do this is to use controllers and have  home page controller  (? or path?) and sign in controller and sign up contorller. so when program is run, we call home page controller.execute, which does: if user enters 1 program does signupcontroller.execute. if user enters 2 program does sign in controller.execute. then say user enters 2. The program does signincontroller.execute. in the sign in controller, it asks for user name and password, and signupcontroller creates the signininputdata and pass it thru the signininteractor.execute, if unsuccessful we execute home page controller again so it goes back to the home page. this way the flow is dynamic instead of being fixed. 
But i realized as ive finsihed writing this that we dont really need to do this since we dont need to "go back" to a previous stage, and our code for tying everythiing together doesnt need to be extendable since it's just to showcase our use cases. I dont want to delete what i've written since i spend like 10 mins typing all this out so i will keep it here but I think what's here is good, but my proposal might be worth considering since it would follow clean architecture principles maybe